docs/release-notes/2.25.0.md

```diff
@@ -0,0 +1,78 @@
+# Percona Monitoring and Management 2.25.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">December 7, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## Release Highlights 
+- **Percona Platform (Technology Preview):** 
+    - Ability to connect your Percona Monitoring and Management instances to your Percona Platform to get the latest intelligence from Percona and have all your accounts and relations with Percona centralized. READ_MORE
+
+- **Enhanced PostgreSQL monitoring**
+    - From this release, PMM will let users who add PostgreSQL Servers be monitored by PMM to [specify database name for monitoring](https://deploy-preview-643--pmm-doc.netlify.app/details/commands/pmm-admin.html#postgresql). Previous versions have the default `postgres` database, which wasn't always convenient for the users. 
+    - This Release of PMM is supported and compatible with the recent release of the PostgreSQL extension - `pg_stat_monitor` v1.0.0. You can read more about its features and advantages in this blog post (LINK to pg_stat_monitor release announcement or Userguide)
+    - PMM 2.25 also be compatible and working fine with the latest Percona Disctibutions for PostgreSQL released today (LINK), which includes the newest version of pg_stat_monitor extension.  

```

Comment:
```suggestion
    - PMM 2.25.0 is also now compatible with the latest Percona Distribution for PostgreSQL released today (LINK), which includes the newest version of pg_stat_monitor extension.  

```

Reply:
just be aware that "released today" may not be accurate when we actually release ;-) might be better to just call out the specific distribution version so that in 2 months someone reading this doesn't jump to the wrong conclusion